internal/k8s/statusaddress_test.go

```diff
@@ -0,0 +1,355 @@
+// Copyright © 2020 VMware
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8s
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/projectcontour/contour/internal/assert"
+	"github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestServiceStatusLoadBalancerWatcherOnAdd(t *testing.T) {
```

Comment:
Should make this table style tests like in `TestStatusAddressUpdater_OnUpdate`.

Reply:
The `ServiceStatusLoadBalancerWatcher` tests are actually old, I've moved them into a better-named file, so they are showing as new. I've kept the same format for the `ServiceAddressUpdater` tests, so all the tests in the file were consistent. This is also the style of tests that @jpeach moved the `StatusClient` tests to, I think.